saleor/graphql/core/connection.py

```diff
@@ -1,11 +1,227 @@
-from typing import Union
+from typing import Any, Dict, List, Tuple, Union
 
 import graphene
 import opentracing as ot
-from django.db.models import Manager, QuerySet
-from graphene import Field, List, NonNull, ObjectType, String
+from django.db.models import Manager, Model as DjangoModel, Q, QuerySet
 from graphene.relay.connection import Connection
 from graphene_django_optimizer.types import OptimizedDjangoObjectType
+from graphql_relay.connection.connectiontypes import Edge, PageInfo
+from graphql_relay.utils import base64, unbase64
+
+from ..core.enums import OrderDirection
+
+ConnectionArguments = Dict[str, Any]
+
+
+def to_global_cursor(values):
+    if not isinstance(values, list):
+        values = [str(values)]
+    else:
+        values = [str(value) for value in values]
+    return base64("::".join(values))
+
+
+def from_global_cursor(cursor) -> List[str]:
+    values = unbase64(cursor)
+    values = values.split("::")
+    if isinstance(values, list):
+        values = [None if value == "None" else value for value in values]
+        return values
+    if values == "None":
```

Comment:
I think it will make debugging and working with this harder

if I have sorting fields `[name, slug, pk]`
I should have "unpacked" cursor as `[name, slug, pk]`

if I have name == None and didn't returned value I would end up with different cursor length than sorting_fields length

I will be impossible to create filter out of this

Reply:
and None in filter is value as any other

Reply:
`None` is a Python literal but also a perfectly valid string, what if I name my product "None"?

Reply:
@patrys damn it :(

I don't have solution for that situation 😭

Reply:
ok I'll try `\N` and will see what happens

Reply:
@patrys proposed `base64(json.dumps([none, 1, "123"]))` and it look like it's working, perfect